### ISSUE TITLE
robbing

### ISSUE THREAD

**Issue by CREASE-C13** [ChainedDev/CityLife#675]
Alright, I think this is a bit ridiculous about the robbing rule obviously I follow it but its like it says you need a good reason to rob someone but then now it's pretty much impossible to rob someone unless you're having gang beef or other occurrences like that cops are pretty much immune to getting robbed if you rob a cop they tell you to join wfs and then a person that's normally part of sasp decides if it's a good enough reason, to be honest, this suggestion is stupid because it's pretty much always been like this but the reason you rob someone is for there items it makes sense it's not like in real life if I decided to do crime I'm gonna rob you for the fun of it I'm robbing you for your items to make money same thing with cops I think if you get caught robbing a cop then you should get like a 150k fine or 100k fine to stop people doing it in the future.  And this thing with recognizing voices is ridiculous if I knew you in the city and I have talked to you a lot of times and you rob me with your normal voice you should be able to know it's them just like you would know it's them irl to make the Rp more immersive let them mask their voice with a voice changer or let them put on a voice. It seems to me like the Sasp never want to take an L and never will if I get away with a crime it doesn't affect them but if I get arrested it affects me a lot like it seems like the city is set up so the cops don't take Loses but just my opinion. and please don't put the clown emotes or some stupid shit reply to this and tell me what you think instead of being a pussy.

**Comment by xrowdydawy** [comment]
Robbing people just for the monetary gain do not create good rp-scenarios whatsoever. If you want to roleplay as a crim, there are plenty of other things you can do instead of robbing random people just for some items. Robbing for monetary gain is just overall a horrible roleplay experience for the parties involved, that is at least my opinion.

**Comment by CREASE-C13** [comment]
i mean there's a lot of successful servers that have this where you can rob for monetary gain i mean maybe civs no since this is not whitelisted but cops with the right consequences would be fine

**Comment by xrowdydawy** [comment]
Why don't you play on those other successful servers instead then?

**Comment by CREASE-C13** [comment]
this is a suggestion i played this server for a year i hardly fly in anymore because this server has so many rules and i want it to improve because i like the community here but this server is pretty much monitory gained based and not rp which is dumb as fuck and i wish it was whitelisted

**Comment by CREASE-C13** [comment]
and there are so many limitations to what you do like you can't create a good RP scene without getting fucked by rules it seems like a lot of creativity is limited

**Comment by MiGrane4Mayor** [comment]
We allowed cops to get robbed for a while but it always gets abused. Traffic stop rp with someone (nothing crazy regular ticket and go) 4 wanna be thugs pull up on motorcycles (officer get your hands up) there's no rp to that so it forces the staff to shut the whole rule down when people can't really do it for the right reasons. It's not like that one officer was followed and targeted for a hate crime against a gang member weeks ago. Instead it's literally (oh shit a cop by himself let's get that mf) you got to see both sides to the argument. Being a very popular public server means we get loads of kids who just think it's legit a modded gta. It's more of a "we had to adapt to the way people are rping" kinda rule i think.
